.ladle/adapters/PlainComponentAdapter.tsx

```diff
@@ -63,6 +64,17 @@ export const PlainComponentAdapter: ComponentsContextType = {
       </div>
     )
   },
+  Breadcrumbs: ({ breadcrumbs, currentBreadcrumbId, ariaLabel, onClick }: BreadcrumbsProps) => {
+    return (
+      <div className="progress-breadcrumbs">
```

Comment:
Would you mind doing this with a nav like wai aria shows? Should be a simple change https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/examples/breadcrumb/